src/precice/bindings/python_future/test/SolverInterface.cpp

```diff
@@ -186,7 +186,11 @@ void SolverInterface:: getMeshVertexIDsFromPositions
   int           size,
   const double* positions,
   int*          ids ) const
-{}
+{
+  for(int i = 0; i < size; i++){
```

Comment:
You could use the infamous [`std::iota`](https://en.cppreference.com/w/cpp/algorithm/iota) here. :wink: 
`std::iota(ids, ids+size, 0)`

Reply:
Done: https://github.com/precice/precice/pull/496/commits/0eafc785ff4dcff403823a74a69ce82fea585520